casepro/msgs/forms.py

```diff
@@ -79,3 +79,65 @@ def clean_keywords(self):
     class Meta:
         model = Label
         fields = ('name', 'description', 'is_synced', 'keywords', 'groups', 'field_test', 'ignore_single_words')
+
+
+class LanguageForm(forms.ModelForm):
+
+    code = forms.CharField(label=_("Code"), max_length=6,
```

Comment:
Having users enter ISO codes manually doesn't seem ideal. Would seem better to let them select languages by name using autocomplete if list is too long for a dropdown.